apps/backend/.env.example

```
@@ -0,0 +1,3 @@
+SUPABASE_URL=http://127.0.0.1:54321
+SUPABASE_SERVICE_ROLE_KEY=your-service-role-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_SERVICE_ROLE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)